cl/search/migrations/0090_update_opinion_sha1.sql

```diff
@@ -0,0 +1,7 @@
+BEGIN;
+--
+-- Alter field sha1 on opinion
+--
+ALTER TABLE "search_opinion" ALTER COLUMN "sha1" SET DEFAULT '';
+ALTER TABLE "search_opinion" ALTER COLUMN "sha1" DROP DEFAULT;
+COMMIT;
```

Comment:
This has no effect on the DB, so we shouldn't have a sql file for this one, right? It's OK to lack a `.sql` file for one of our "migrations" if it's not a migration of the DB at all (as in this instance). I'm going to add a comment to MIGRATING.md about migrations that are noops being more trouble than they're worth. Gotta beef up that file.

Reply:
This has been removed